modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/UnionPlannerTest.java

```diff
@@ -93,4 +167,14 @@ private static UnaryOperator<TableBuilder> createTestTable(String tableName) {
                 .addColumn("NAME", NativeTypes.STRING)
                 .addColumn("SALARY", NativeTypes.DOUBLE);
     }
+
+    private Predicate<? extends RelNode> projectFromTable(String tableName, String... exprs) {
```

Comment:
It might be worth moving this method to `AbstractPlannerTest` and removing the duplicate from `SetOpPlannerTest`.

Reply:
Fixed. Thanks.